dev-libs/libnl/files/0003-route-add-missing-rtnl_nh_get_oif-symbol.patch

```diff
@@ -0,0 +1,34 @@
+From 470c0a62b8b26d7dcf553f859a7abcd62d6b0890 Mon Sep 17 00:00:00 2001
```

Comment:
As I've said before, we cannot do this. See https://github.com/gentoo/gentoo/pull/42116#discussion_r2114077465 please.